server/src/services/users.service.ts

```
@@ -1,16 +1,28 @@
 import { ILoginInfo, IUser } from '../interfaces/users.interface';
 import userModel from '../models/user.Model';
 import validator from 'validator';
+import nodemailer from 'nodemailer';
 import argon2 from 'argon2';
 import jwt from 'jsonwebtoken';
 import crypto from 'crypto';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)